sfa_dash/blueprints/admin.py

```diff
@@ -72,7 +72,7 @@ def flatten_reports(self, report_list):
         """
         reports = [report.copy() for report in report_list]
         for report in reports:
-            report.update(report.get('report_parameters'))
+            report.update(report.pop('report_parameters'))
```

Comment:
do we need the raw_report for anything here? good to drop if we don't

Reply:
Actually, now I don't understand the purpose of this function for the admin pages. What parameters of a report are necessary besides the name, provider, and id?

Reply:
It's really just moving all of the keys in report parameters to the top level of the report metadata dictionaries. You're right that most aren't necessary, this could be reduced to just move the name field.